hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/SelectQueryBuilder.java

```diff
@@ -65,7 +65,7 @@ class SelectQueryBuilder {
             sb.append("*");
         }
         sb.append(" FROM ")
-          .append(table.getExternalName());
+          .append(table.getExternalName()[0]);
```

Comment:
we use quoteIdentifier elsewhere (eg. in `MySQLUpsertQueryBuilder`). why not here?

Reply:
#24201

Reply:
Addressed in a followup PR 

https://github.com/hazelcast/hazelcast/pull/24201/files#diff-2af909dc378ab9aa63c46547537f1173a1b84c56e9a18854b144bae058c3a874R50